trackbasket_be/models/krogerservice.py

```diff
@@ -47,4 +48,47 @@ def closest_store(cls, zipcode):
               'latitude': parsed_response['data'][0]['geolocation']['latitude'],
               'longitude': parsed_response['data'][0]['geolocation']['longitude']            
             }
+  
+  @classmethod
+  def item_search(cls, term, at_risk_user_id):
+    
+    user = AtRiskUser.find_by_id(at_risk_user_id) 
+    nearest_store = user.store[0]
+    nearest_store_id = nearest_store.location_id
+    parameters = {'filter.term': term, 'filter.locationId': nearest_store_id}
+    headers = {'Authorization': 'Bearer {}'.format(Krogerservice.return_token())}
+    response = requests.get('https://api.kroger.com/v1/products', params=parameters, headers=headers)
+
+    json_items = response.json()['data']
+    
+    items = []   
+      
+    for item in json_items:
+      data = {
+        
+              "name":None, 
+              "description":None, 
+              "unit_price":None, 
+              "image":None, 
+              "productId":None, 
+              "upc":None, 
+              "aisle_number":None
+        }
+      
+      data['name'] = item['description']
+      data['productId'] = item['productId']
+      data['upc'] = item['upc']
+      
+      for price in item['items']:
+        data['unit_price'] = price['price']['regular']
+        
+      for aisle in item['aisleLocations']:
+        data['aisle_number'] = int(aisle['number']) 
```

Comment:
maybe we could remove the `for` loop here because it seemed when we checked this morning that there was only one element in the array